template/src/components/ui/ConfirmationDialog.tsx

```diff
@@ -36,7 +36,7 @@ export const ConfirmationDialog = ({
     hideCancel?: boolean;
 }) => {
     return (
-        <CustomDialog open={open}>
+        <CustomDialog open={open || false}>
```

Comment:
warum genau `|| false`?

Reply:
Schließe mich an -> wenn das `open` prop im `CustomDialog` korrekt mit `?` getyped ist entspricht das einem `false`.
Booleans, wenn sie nicht explizit `false` sein müssen, immer optional typen

Reply:
Ist in MUI leider getyped als `open: boolean;`. Sonst bin ich voll eurer Meinung.

Reply:
ah k, ich mach dann immer `!!open`oder sauberer wsch. `open ?? false`